staff/mauri-velez/baby-steps/calculate.js

```diff
@@ -0,0 +1,30 @@
+// // /**
```

Comment:
correct this js-doc comment. set it right, like shown in the original description https://pluscoders.com/web-dev/challenges/calculator-app/